mod/photos.php

```diff
@@ -1217,6 +1200,20 @@ function photos_content(App $a)
 			return;
 		}
 
+		if ($cmd === 'drop') {
```

Comment:
Seems like the same command as above. What about creating a small method `photos_drop_confirm`. Because this would avoid the risk that only one place would get altered in case of changes

Reply:
One is for albums, the other is for photos, the `confirm.tpl` template already is the lowest common denominator.

Reply:
ok